### DISCUSSION TITLE
Web Builder v0.9f and MKS-TinyBee 1.0

### DISCUSSION THREAD

**Discussion by LEOOZERAN** [grblHAL/core#433] [General]
In the Web Builder v0.9f program, the MKS-TinyBee 1.0 board firmware does not work. FOR a simple ESP32. In the ioSender program. Reports a board error (it doesn’t see it) In other programs, there’s also an error (it doesn’t see the board)

**Comment by terjeio** [comment]
Works for me when flashed to a dev board, compiled with default options.
Which error is reported?

**Reply by LEOOZERAN** [reply]
I just tried again. Board only with SD. In the new version of .iO Sender. And..O..MIRACLE! It works! Thank you for answering questions here. Greetings from Ukraine.
